localstack/utils/analytics/__init__.py

```diff
@@ -10,3 +11,7 @@ def _create_global_analytics_bus():
 
 
 log = EventLogger(handler=_create_global_analytics_bus(), session_id=get_session_id())
+
+# our own "GlobalAnalyticsBus"
+usage_event_handler = UsageEventHandler()
+usage_log = UsageLogger(handler=usage_event_handler)
```

Comment:
how does the usage logger compare to the `EventLogger`? is this an orthogonal or a competing construct? when should i use one or the other?

Reply:
It doesn't actually need to be a separate logger since it shouldn't ever be used directly. Only the counters or other similar constructs should be used here :thinking: 

So I could image just removing it here and keeping it only as a local concept in `localstack/utils/analytics/usage.py`, that way there shouldn't be any confusion hopefully.

If you meant this in a more general sense, I'd say anything that you want to use aggregations with and where we are OK with missing out on some values potentially, should use these constructs.